app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -197,37 +233,61 @@ public void close() {
         database.close();
     }
 
+    /**
+     * Method for reading data from shapes table
+     * @return Cursor
+     */
     public Cursor getShapeValues() {
         return this.database.query(DBConstants.DB_TABLE_SHAPE, new String[]{
                         DBConstants.KEY_X, DBConstants.KEY_Y, DBConstants.KEY_ROOM_ID, DBConstants.KEY_MAP_ID, DBConstants.KEY_ID},
                 null, null, null, null, null);
     }
 
+    /**
+     * Method for reading data from doors table
+     * @return Cursor
+     */
     public Cursor getDoorValues() {
         return this.database.query(DBConstants.DB_TABLE_DOORS, new String[]{DBConstants.KEY_CONNECTED_TO,
                         DBConstants.KEY_MAP_ID, DBConstants.KEY_ROOM_ID, DBConstants.KEY_ID, DBConstants.KEY_X, DBConstants.KEY_Y},
                 null, null, null, null, null);
     }
 
+    /**
+     * Method for reading data from QRs table
+     * @return Cursor
+     */
     public Cursor getQrValues() {
         return this.database.query(DBConstants.DB_TABLE_QRS, new String[]{DBConstants.KEY_ID,
                 DBConstants.KEY_INFO, DBConstants.KEY_X, DBConstants.KEY_Y, DBConstants.KEY_MAP_ID,
                 DBConstants.KEY_ROOM_ID}, null, null, null, null, null);
     }
 
+    /**
+     * Method for reading data from museums table
+     * @return Cursor
+     */
     public Cursor getMuseumValues() {
         return this.database.query(DBConstants.DB_TABLE_MUSEUMS, new String[]{DBConstants.KEY_ID,
                 DBConstants.KEY_DESCRIPTION, DBConstants.KEY_LOCATION, DBConstants.KEY_MAP_STATUS,
                 DBConstants.KEY_NAME, DBConstants.KEY_MAP_SIZE, DBConstants.KEY_URL}
                 , null, null, null, null, null);
     }
 
+    /**
+     * Method for reading data from rooms table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.